ekg-core.cabal

```diff
@@ -28,10 +28,10 @@ library
     System.Metrics.ThreadId
 
   build-depends:
-    ghc-prim < 0.4,
-    base >= 4.5 && < 4.8,
+    ghc-prim < 0.5,
+    base >= 4.5 && < 4.10,
```

Comment:
Isn't that bound what allows it to be built against HEAD? I guess you don't want something to be "compatible" but not eventually with the final version.